### PR TITLE
Add 0.7 release notes

### PR DESCRIPTION
**Pull request by yuzelin**

### CODE REVIEW

File: pages/content/releases/release-0.7.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+In this version, we mainly focus on enhancement and optimization for the existing features. For more details, please 
+refer to the following content below.
+
+## Flink
+
+### Lookup Join
+
+1. Fix bug that lookup join cannot handle sequence field of dim table.
+2. Introduced primary key partial lookup based on Paimon hash lookup for lookup join. 
+3. Use parallel reading and bulk loading to speed up initial data loading of dim table.
```
Comment (Aitozi) [inline pages/content/releases/release-0.7.md:43]: Should we mention the new lookup query service here ?

Comment (JingsongLi) [reply pages/content/releases/release-0.7.md:43]: Not work for 0.7...

**Review by JingsongLi — APPROVED** [review]
+1

File: pages/content/releases/release-0.7.md (hunk 1 above)
Comment (Aitozi) [reply pages/content/releases/release-0.7.md:43]: Oh..

**Pull request merged**